eQuiz/modules/eQuiz.Web/Areas/Admin/Views/Default/Index.cshtml

```diff
@@ -1,16 +1,45 @@
-﻿
-@{
-    ViewBag.Title = "Index";
-    Layout = "~/Views/Shared/Layout.cshtml";
-}
-
-@section HeadStyles 
-{    
-    <link href="~/Areas/Admin/Styles/equiz-temp.css" rel="stylesheet" />
-}
-
-<h2>Hello Admin!!!</h2>
-
-<div>
-    <a href="@Url.Action("Index", "Home", new { area = "" })">Back Home</a>
-</div>
\ No newline at end of file
+﻿@{
+    Layout = "~/Views/Shared/_Layout.cshtml";
+}
+
+<div ng-app="equizModule">
+    <div ng-controller="LayoutController as lc">
+        <div layout="row" style="padding:10px">
+            <div>
+                <button style="text-decoration:solid; font-size:20px" ng-click="lc.GoToReview()">
+                    Students
+                </button>
+                <button style="text-decoration:solid; font-size:20px">
+                    Quizzes
+                </button>
+                <label style="text-decoration:solid; font-size:20px; float:right">
+                    Full Name
+                </label>
+            </div>
+        </div>
+        <div compile="lc.review"></div>
+    </div>
+</div>
+
+@section FootScripts
+{
+    <script src="~/Areas/Admin/Scripts/shared/equiz.module.js"></script>

```

Comment:
Why do we need this file?

That mode is already created in the common area - "Scripts/app/shared/equiz.module.js"